pkg/api/validation/validation.go

```diff
@@ -1955,6 +1966,9 @@ func ValidatePodSecurityContext(securityContext *api.PodSecurityContext, spec *a
 				allErrs = append(allErrs, field.Invalid(fldPath.Child("supplementalGroups").Index(g), gid, msg))
 			}
 		}
+		if securityContext.AppArmorProfile != "" {
+			allErrs = append(allErrs, validateAppArmorProfile(securityContext.AppArmorProfile, fldPath.Child("appArmorProfile"))...)
```

Comment:
Compare with https://github.com/kubernetes/kubernetes/pull/28300#issuecomment-235143192 concerning the field path. We should decide for one way or the other.